apps/bfd-server/bfd-server-war/src/test/resources/openapi/headers.yaml

```diff
@@ -0,0 +1,162 @@
+        This ID SHALL be included in all log events for the frontend system. This allows requests to be 
+        traced across systems. For asynchronous jobs, users SHALL use the BlueButton-OriginalQueryId for 
+        the HTTP request that created/submitted/whatever the job.
+    BlueButton-OriginalQueryCounter-header:
+      in: header
+      name: BlueButton-OriginalQueryCounter
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+      description: |
+        Start at 1 and increment for every request to BFD with the same BlueButton-OriginalQueryId value.
+    BlueButton-OriginalQueryTimestamp-header:
+      in: header
+      name: BlueButton-OriginalQueryTimestamp
+      required: false
+      schema:
+        type: string
+        format: date-time
+      description: | 
+        An ISO-8601 UTC timestamp representing (roughly) when the original request reached the frontend system.
+    BlueButton-DeveloperId-header:
+      in: header
+      name: BlueButton-DeveloperId
+      required: true
+      schema:
+        type: string
+      description: | 
+        The unique ID in the frontend system for the developer/vendor of the third-party application.
+    BlueButton-Developer-header:
+      in: header
+      name: BlueButton-Developer
+      required: false
+      schema:
+        type: string
+      description: |
+        The human-readable name in the frontend system for the developer/vendor of the third-party application.
+    BlueButton-ApplicationId-header:
+      in: header
+      name: BlueButton-ApplicationId
+      required: true
+      schema:
+        type: string
+      description: |
+        The unique ID in the frontend system for the application/client that will 
+        receive the data.
+    BlueButton-Application-header:
+      in: header
+      name: BlueButton-Application
+      required: false
+      schema:
+        type: string
+      description: | 
+        The human-readable name in the frontend system for the application/client that will 
+        receive the data.
+    BlueButton-UserId-header:
+      in: header
+      name: BlueButton-UserId
+      required: true
+      schema:
+        type: string
+      description: | 
+        The unique ID in the frontend system for the user (e.g. beneficiary) that data is being requested 
+        on behalf of.
+    BlueButton-User-header:
+      in: header
+      name: BlueButton-User
+      required: false
+      schema:
+        type: string
+      description: | 
+        The human-readable login ID (e.g. email address) in the frontend system for the user (e.g. beneficiary) 
+        that the data is being requested on behalf of.
+    BlueButton-BeneficiaryId-header:
+      in: header
+      name: BlueButton-BeneficiaryId
+      required: false
+      schema:
+        type: string
+      description: |
+        The unique beneficiary ID (e.g. Patient.id) from BFD for the user the beneficiary 
+        whose data is being requested.
```

Comment:
If I were a consumer, I'd want to know why I'd need to use this header.. given that its not required and wed generally pass this in as a parameter if we were searching on bene id. (I don't actually know what this header is used for offhandedly)

Reply:
🤔

Reply:
I can remove references to this one if it's not being used.